natchez-extras-http4s/src/main/scala/com/ovoenergy/natchez/extras/http4s/client/TracedClient.scala

```diff
@@ -24,7 +25,11 @@ object TracedClient {
   private def trace[F[_]]: F ~> Traced[F, *] =
     Kleisli.liftK
 
-  def apply[F[_]: Sync](client: Client[F], config: Configuration[F]): TracedClient[Traced[F, *]] =
+  def apply[F[_]: Sync](
+    client: Client[F],
+    config: Configuration[F],
+    removeNumericPathSegments: Uri => String = removeNumericPathSegments
+  ): TracedClient[Traced[F, *]] =
```

Comment:
Currying seems a more elegant solution here but if I do
`(.....)(removeNumericPathSegments: Uri => String = removeNumericPathSegments)`
the tests fail

because the method is not already curried.

Reply:
This will break the backward compatibility.